app/controllers/api/v1/users_controller.rb

```diff
@@ -29,6 +29,57 @@ def update
         end
       end
 
+      # exist
+      #
+      # When given a user's email, checks if that user is already registered in the database
+      # and sends the proper redirect url to the frontend accordingly. Called when logging in
+      # via social media.
+      # @param params The data passed in from the frontend, from which [:user][:email] (the user's email) is extracted. Expecting :user in params with parameter :email inside.
+
+      def exist
```

Comment:
Yeah, Yard documentation takes a while to get the hang of.  For this particular method, perhaps something like:

```ruby
      # For social media logins, renders the appropriate `redirect_to` path.
      #
      # @param params [Hash] ActionController::Parameters that require a :user key, with a nested :email key.
      #   For example: { user: { email: "john@example.com" } }
      # @return [String] A string of the user's redirect_to path
      #
      def exist
        ...
      end
```

Reply:
done.